src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -134,6 +135,10 @@ object EasyStageDataset {
       } yield ()
     }
 
+    def isRidirecting(bagRelativePath: String): Try[Boolean] = {
+      Success(s.stageFileDataAsRedirectDatastreams) || (Success(s.stubAVfiles) && isAVType(bagRelativePath))
```

Comment:
happy with how these logical operators turned out! However, I thought you were going to write it without the explicit `Success` wrappings. We had the alternatives like `Boolean && Try[Boolean]` before, right?